app/controllers/users/dossiers_controller.rb

```diff
@@ -18,15 +18,9 @@ class DossiersController < UserController
     def index
       @user_dossiers = current_user.dossiers.includes(:procedure).order_by_updated_at.page(page)
       @dossiers_invites = current_user.dossiers_invites.includes(:procedure).order_by_updated_at.page(page)
-
-      @current_tab = current_tab(@user_dossiers.count, @dossiers_invites.count)
-
-      @dossiers = case @current_tab
-      when 'mes-dossiers'
-        @user_dossiers
-      when 'dossiers-invites'
-        @dossiers_invites
-      end
+      @dossiers_supprimes = DeletedDossier.where(user_id: current_user.id).order_by_updated_at.page(page)
+      @statut = params[:statut]
+      @statut.blank? ? @statut = 'mes-dossiers' : @statut = params[:statut]
```

Comment:
```suggestion
      @statut = params[:statut].blank? ? 'mes-dossiers' : params[:statut]
```

je pense que tu pourrais en faire une méthode privée `def statut`

Je me demande également s'il ne faudrait pas être plus précautionneux avec ce paramètre usager en faisant une comparaison explicite : `case params[:statut] when ...`